css/style.css

```diff
@@ -0,0 +1,135 @@
+body {
+    font-size: 18px;
+    font-family: "Open Sans", Arial, sans-serif;
+
+}
+
+
+.wrapper { 
+    max-width: 1600px;
+    margin: 0 auto;
+}
+
+.social_link {
+    position: relative;
+    top: 30px;
+    right: 215px;
+    color: #FFFFFF;
+    float: right;
+    margin-right: 15px;
+}
+
+.social a:hover{
+    color: #03204f;
+    border-radius: 2px;
+}
+
+.banner {
+    padding: 174px 215px;
+    background: url("../img/banner.png") no-repeat center top;
+}
+
+.banner_info h1,h2 {
+     width: 800px;
+     height: 55px;
+     font-family: Open Sans;
+     font-style: normal;
+     color: #fff;
+     font-weight: normal;
+     line-height: 66px;
+     font-size: 44px;
+     letter-spacing: 0.016em;
+    }
+
+    .btn {
+        font-family: "Open Sans";
```

Comment:
Again no needs in specifying it again since we already have it described for body